examples/js/loaders/MTLLoader.js

```diff
@@ -12,7 +12,7 @@ THREE.MTLLoader = function ( manager ) {
 
 THREE.MTLLoader.prototype = {
 
-	constructor: THREE.MTLLoader,
+	constructor: MTLLoader,
```

Comment:
I think this will fail – `MTLLoader` is not defined as a variable.

Reply:
Well, maybe not fail, but it is undefined...

Reply:
You can leave this as `THREE.MTLLoader`, and we'll fix the issue with the module converter script separately.

Reply:
Yep your right, it worked as a module but not on the web example. It requires MTLLoader to be refactored, I've done that now - but I dislike doing a PR where I basically mod every line of a file :(

If you're fixing in a different way thats fine, but here's my solution if anyone else needs it.

Thanks!